MAINTAINERS.yml

```
@@ -259,6 +259,20 @@ Ambiq Platforms:
   labels:
     - "platform: Ambiq"
 
+BeagleBoard Platforms:
+  status: maintained
+  maintainers:
+    - jadonk
+  collaborators:
+    - ayush1325
+    - con-pax
+    - vaishnavachath
+  files:
+    - boards/arm/beagle*
+    - boards/riscv/beagle*
+  labels:
+    - "platform: BeagleBoard Platforms"
```

don't think any other platform label has "... Platforms" in them, hence why my previous suggestion :) 

```suggestion
    - "platform: BeagleBoard"
```